<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>678</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QT音乐播放器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="m_mainWidget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <item>
        <widget class="QWidget" name="m_displayWidget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="QPushButton" name="m_serachButton">
          <property name="geometry">
           <rect>
            <x>950</x>
            <y>20</y>
            <width>81</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border:1px solid;
	border-radius:10px;
	border-color: rgb(129, 129, 129);
}
QPushButton:hover {
	background-color: lightgray;
}
QPushButton:pressed {
	background-color: gray;
}
</string>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
         <widget class="QTableView" name="m_musicTable">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>80</y>
            <width>521</width>
            <height>192</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
         <widget class="QListWidget" name="m_lyricList">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>280</y>
            <width>521</width>
            <height>192</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QListWidget{
	border: transparent;
	border-radius: 5px;
	box-shadow: 3px 3px 5px rgba(0, 0, 0, 0.2);
}

QListWidget::item {
	font-size: 20px;
	text-align: center;
}</string>
          </property>
         </widget>
         <widget class="QSlider" name="m_verticalSlider">
          <property name="geometry">
           <rect>
            <x>970</x>
            <y>360</y>
            <width>21</width>
            <height>111</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:vertical
{ 
	border: 1px solid rgb(0, 0, 0); 
	width: 2px; 
	margin: 0px 0;
	top: 5px; 
	bottom: 5px; 
}  

QSlider::handle:vertical
{
	border: 0px ;	
	image: url(:/image/circle.png);
	width: 17px; 
	margin: -8px -8px -8px -8px; 
}

QSlider::sub-page:vertical
{
	background-color: gray;
}

QSlider::add-page:vertical
{
	background-color: rgb(20,20,20);
}

</string>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="m_musicName">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>400</y>
            <width>431</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">
QLabel{
	border-width:0px;
	font: 20pt &quot;华文行楷&quot;;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="m_searchMusic">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>20</y>
            <width>431</width>
            <height>51</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
         <widget class="QWidget" name="m_musicWidget" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>480</y>
            <width>1011</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,2,1,0,10,1,2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="m_lastMusic">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	image: url(:/image/lastsong.png);
	background-color: rgb(255, 255, 255);
	border:1px solid;
	border-radius:10px;
	border-color: rgb(129, 129, 129);
}
QPushButton:hover {
	background-color: lightgray;
}
QPushButton:pressed {
	background-color: gray;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_play">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	image: url(:/image/play.png);
	background-color: rgb(255, 255, 255);
	border:1px solid;
	border-radius:10px;
	border-color: rgb(129, 129, 129);
}
QPushButton:hover {
	background-color: lightgray;
}
QPushButton:pressed {
	background-color: gray;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="shortcut">
              <string/>
             </property>
             <property name="autoRepeatDelay">
              <number>300</number>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_nextMusic">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	image: url(:/image/nextsong.png);
	background-color: rgb(255, 255, 255);
	border:1px solid;
	border-radius:10px;
	border-color: rgb(129, 129, 129);
}
QPushButton:hover {
	background-color: lightgray;
}
QPushButton:pressed {
	background-color: gray;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="m_currentTime">
             <property name="styleSheet">
              <string notr="true">QLabel{
	font: 9pt &quot;黑体&quot;;
	border-width:0;
}</string>
             </property>
             <property name="text">
              <string>00:00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="m_horizontalSlider">
             <property name="styleSheet">
              <string notr="true">QSlider::groove:horizontal 
{ 
	border: 1px solid rgb(0, 0, 0); 
	height: 2px; 
	margin: 0px 0;
	left: 5px; 
	right: 5px; 
}  

QSlider::handle:horizontal
{
	border: 0px ;	
	image: url(:/image/circle.png);
	width: 17px; 
	margin: -8px -8px -8px -8px; 
}

QSlider::add-page:horizontal
{
	background-color: gray;
}

QSlider::sub-page:horizontal
{
	background-color: rgb(20,20,20);
}

</string>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="m_totalTime">
             <property name="styleSheet">
              <string notr="true">QLabel{
	font: 9pt &quot;黑体&quot;;
	border-width:0;
}
</string>
             </property>
             <property name="text">
              <string>00:00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_adjustVoice">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	image: url(:/image/addvoice.png);
	background-color: rgb(255, 255, 255);
	border:1px solid;
	border-radius:10px;
	border-color: rgb(129, 129, 129);
}
QPushButton:hover {
	background-color: lightgray;
}
QPushButton:pressed {
	background-color: gray;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <zorder>m_serachButton</zorder>
         <zorder>m_musicTable</zorder>
         <zorder>m_lyricList</zorder>
         <zorder>m_musicName</zorder>
         <zorder>m_verticalSlider</zorder>
         <zorder>m_searchMusic</zorder>
         <zorder>m_musicWidget</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
